sopel/modules/url.py

```diff
@@ -231,6 +259,17 @@ def get_hostname(url):
     return hostname
 
 
+def get_tinyurl(url):
+    """ Returns a shortened tinyURL link of the URL. """
+    tinyurl = "https://tinyurl.com/api-create.php?url=%s" % url
+    res = requests.get(tinyurl)
+    # Replace text output with https instead of http to make the
+    # result an HTTPS link (the third argument is so that it only
+    # replaces the first HTTP portion of the TinyURL and not, say,
+    # a hypothetical short URL with HTTP in it.
+    return res.text.replace("http", "https")
```

Comment:
You referenced a "third argument" in the comment block, but passed only two arguments to the function. That said, you can make this even safer by replacing `http://` with `https://` instead.

Reply:
Yeah, you're right. This is done.